src/main/resources/application.properties

```diff
@@ -1 +1,9 @@
+api.bungie.address.player=https://www.bungie.net/Platform/Destiny2/SearchDestinyPlayer/-1/{nickname}/
 
+spring.datasource.url=jdbc:mysql://remotemysql.com:3306/T3qRnlKrf3
```

Comment:
I was forced to add these properties. Draft version.

Reply:
👍  in "prod" use environment variable